setup.cfg

```diff
@@ -12,3 +12,7 @@ include_trailing_comma = True
 force_grid_wrap = 0
 use_parentheses = True
 line_length = 88
+
+[options.entry_points]
+console_scripts=
+    wdl-cwl-translator = wdl2cwl.main_oop:main
```

Comment:
I thought it would be useful to have the command in the `$PATH` of the virtual environment to test workflows? I was calling the `main_oop` directly or editing tests. But I think ultimately we want to give a tool to users? WDYT @mr-c ?

Reply:
Can you name the script `wdl2cwl`?